.travis.yml

```diff
@@ -10,13 +10,13 @@ rvm:
 matrix:
   include:
     - rvm: *latest_ruby
-      env: JEKYLL_VERSION=3.7.0
+      env: JEKYLL_VERSION=3.7.0 FARADAY_VERSION=0.17
     - rvm: *latest_ruby
-      env: JEKYLL_VERSION=4.0.0.pre.alpha1
+      env: JEKYLL_VERSION=3.7.0 FARADAY_VERSION=1.0
     - rvm: *latest_ruby
-      env: FARADAY_VERSION=0.17.3
+      env: JEKYLL_VERSION=4.0.0.pre.alpha1 FARADAY_VERSION=0.17
```

Comment:
should be `JEKYLL_VERSION=4.0.0` now

Reply:
@DirtyF This is an outdated diff.. The branch is already set up to test the latest versions.